svg2polylines/src/lib.rs

```diff
@@ -255,37 +242,26 @@ fn parse_path(mut path: path::Tokenizer) -> Vec<Polyline> {
 }
 
 /// Parse an SVG string into a vector of polylines.
-pub fn parse(svg: &str) -> Result<Vec<Polyline>, String> {
-    // Vector that will hold resulting polylines
-    let mut polylines = Vec::new();
-
+pub fn parse(svg: &str) -> Vec<Polyline> {
     // Tokenize the SVG strings into svg::Token instances
-    let mut tokenizer = svg::Tokenizer::from_str(&svg);
+    let tokenizer = Tokenizer::from_str(&svg);
 
     // Loop over all tokens and parse the apths
```

Comment:
Oh, a typo: apths -> paths. If you're at it, could you maybe fix that one too? :)